controllers/tutorialsControllers.js

```diff
@@ -36,6 +38,7 @@ const updateTutorial = async (req, res) => {
     if (typeof title !== "undefined") update.title = title;
     if (typeof description !== "undefined") update.description = description;
     if (typeof published !== "undefined") update.published = published;
+    await schema.validateAsync(req.body);
```

Comment:
Did you test by passing invalid title or any other fields? What's the response you're getting?

Reply:
This is how you should be using Joi to validate:
```
const { error, body: tutorialBody } = postTutorialSchema.validate(req.body);

# Return error response if there is anything in error variable
...
...
...
const tutorial = await Tutorial.findByIdAndUpdate(
      { _id: req.params.id },
      tutorialBody,
      { new: true }
    );
```

Reply:
@jaimin45 , Please follow this example that I shared above.